internal/backend/cache/file.go

```diff
@@ -210,6 +210,9 @@ func (c *Cache) list(t restic.FileType) (restic.IDSet, error) {
 	dir := filepath.Join(c.path, cacheLayoutPaths[t])
 	err := filepath.Walk(dir, func(name string, fi os.FileInfo, err error) error {
 		if err != nil {
+			// ignore ErrNotExist to gracefully handle multiple processes clearing the cache
```

Comment:
Please change the commit description to

`cache: fix race condition in cache cleanup` or similar.